pkg/operator/starter.go

```diff
@@ -112,6 +118,7 @@ func RunOperator(ctx context.Context, kubeconfig *restclient.Config) error {
 	)
 
 	nodeCADaemonController := NewNodeCADaemonController(
```

Comment:
if you pass in an eventRecorder you will be able to set up your unit tests with https://github.com/openshift/library-go/blob/master/pkg/operator/events/recorder_in_memory.go#L32-L34